Remove implicit function declarations incompatible with new GCC.

This is not UNICOS anyway

--- a/configure.ac
+++ b/configure.ac
@@ -485,14 +485,6 @@ AC_CHECK_DECLS([nanosleep],[],[],[#include <time.h>])
 AC_CHECK_DECLS([drand48],[],[],[#include <stdlib.h>])
 AC_CHECK_DECLS([srand48],[],[],[#include <stdlib.h>])
 
-# Cray UNICOS _rtc() (real-time clock) intrinsic
-AC_MSG_CHECKING([for _rtc intrinsic])
-rtc_ok=yes
-AC_LINK_IFELSE([AC_LANG_PROGRAM([[#ifdef HAVE_INTRINSICS_H
-#include <intrinsics.h>
-#endif]], [[_rtc()]])],[AC_DEFINE(HAVE__RTC,1,[Define if you have the UNICOS _rtc() intrinsic.])],[rtc_ok=no])
-AC_MSG_RESULT($rtc_ok)
-
 AC_MSG_CHECKING([whether a cycle counter is available])
 save_CPPFLAGS=$CPPFLAGS
 CPPFLAGS="$CPPFLAGS -I$srcdir/include"
